### PR TITLE
NumberFormatter: updated doc example headers

### PR DESCRIPTION
**Pull request by wescravens**

### CODE REVIEW

File: doc/api/number/number-formatter.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ zhFormatter( 3.141592 );
 // > "三.一四二"
 ```
 
-Controlling digits by specifying integer and fraction digits counts:
+The number of decimal places can be decreased or increased using `minimumFractionDigits` and `maximumFractionDigits`.
```
Comment (jzaefferer) [inline doc/api/number/number-formatter.md]: Maybe add headers to each example?
```md
#### Static Formatter
#### Instance Formatter
#### Customzing fractions
#### Customizing significant decimal digits
#### Formatting percentages
#### Controlling rounding
#### Reusing formatter instance
```
Not sure about "Customzing" and "Controlling", is there something better?

**Comment by rxaviers** [comment]
@wescravens I liked your changes very much.

**Comment by wescravens** [comment]
@jzaefferer @rxaviers I added headers based on the suggestions.  Let me know what you think.

File: doc/api/number/number-formatter.md [hunk 2]
```diff
@@ -94,7 +100,9 @@ Globalize.numberFormatter({ minimumFractionDigits: 2 })( 1.5 );
 // > "1.50"
 ```
 
-Controlling digits by specifying significant digits counts:
+#### Configuring significant decimal places
```
Comment (jzaefferer) [inline doc/api/number/number-formatter.md]: This seems wrong, this isn't about "places", but about "digits", though I don't understand what `minimumSignificantDigits` and `maximumSignificantDigits` actually do. The examples are not helping.

**Comment by jzaefferer** [comment]
Apart from the slightly-off header, this looks great.

File: doc/api/number/number-formatter.md (hunk 2 above)
Comment (wescravens) [inline doc/api/number/number-formatter.md]: Yeah you're right since I think this applies to integers as well.

**Comment by wescravens** [comment]
@jzaefferer @rxaviers Updated the significant digits header

**Comment by rxaviers** [comment]
Thank you @wescravens

**Pull request closed without merging**